root/usr/share/nethesis/NethServer/Language/en/NethServer_Module_Sogo.php

```diff
@@ -0,0 +1,20 @@
+<?php
+$L['Sogo_header'] = 'Configuration';
+$L['Sogo_Title'] = 'SOGo';
+$L['Sogo_Description'] = 'SOGo is the groupware solution';
+$L['status_label'] = 'Enable SOGo';
+$L['Dav_label'] = 'Enable CalDAV and CardDAV';
+$L['MailAuxiliaryUserAccountsEnabled_label'] = 'Allow Users to add other IMAP accounts';
+$L['ActiveSync_label'] = 'Enable Microsoft ActiveSync';
+$L['AdminUsers_label'] = 'Administrators';
+$L['Advanced_label'] = 'Advanced settings';
+$L['VirtualHost_label'] = 'Make SOGo reachable only from this domain name list';
+$L['Notifications_label'] = 'Notifications';
+$L['ACLs_label'] = 'Sent by emails to involved users of a calendar/address book';
+$L['Folders_label'] = 'Sent by emails on the creation of a calendar/address book.';
+$L['Appointment_label'] = 'Sent by emails to meeting participants';
+$L['EMail_label'] = "Sent by emails to user's events and tasks";
```

Comment:
Can not find this in the SOGo manual,
and also rephrasing as above does not make clear when this notification is sent..
`Send email to user's events and tasks`

Reply:
this comes from `SOGoEnableEMailAlarms` in `/etc/e-smith/templates/etc/sogo/sogo.conf/30notifications` although even the sogo documentation is not clear, do you have a suggestion @markVnl

Reply:
got it :+1: 


SOGoEnableEMailAlarms | Parameter used to enable email-based alarms on events and tasks.   Defaults to NO when unset.   For this feature to work correctly, one must also set the OCSEMailAlarmsFolderURL parameter and enable the associated cronjob. See the Cronjob — EMail reminders section from this document for more information.
-- | --

Reply:
proposition : send email-based alarm on events and tasks

Reply:
none would expect the behavior is to get notified by an email is a task expires.
Will try to find this out..

_if so_ proposition :

`Send email to user for expired events and tasks`